Client/Frontend/Browser/LoginsHelper.swift

```diff
@@ -81,31 +83,43 @@ class PasswordHelper: BrowserHelper {
         return attr
     }
 
-    private func setPassword(password: Password) {
-        if password.username.isEmpty || password.password.isEmpty {
+    private func setCredentials(login: LoginData) {
+        log.debug("Set credential \(login)")
```

Comment:
Without a `description`, I don't know if this logging is worthwhile. With a `description`, I don't know if it's privacy-respecting…